team-agreements/working-agreements/readme.md

```diff
@@ -24,7 +24,7 @@ The following are examples of sections and points that can be part of a working
 
 - Our office hours, when we can expect to collaborate via Microsoft Teams, phone or face-to-face are Monday to Friday 10AM - 5PM
 - We are not expected to answer emails past 6PM, on weekends or when we are on holidays or vacation.
-- We work in different time zones and respsect this, especially when setting up recurring meetings.
```

Comment:
Thanks for the contribution.

I recently submitted PR #351 where this fix is already included. Would you mind reviewing that instead and perhaps abandoning this PR to avoid conflicts. I added you as a reviewer on that PR, if you have the time it would be much appreicated

Reply:
Update, I merged PR #351 this morning